cluster/addons/metadata-proxy/metadata-proxy-configmap.yaml

```diff
@@ -0,0 +1,60 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: metadata-proxy-config
+  namespace: kube-system
+  labels:
+    addonmanager.kubernetes.io/mode: EnsureExists
+data:
+  nginx.conf: |-
```

Comment:
I think proxy_cache is off by default.

http://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_cache